tests/unit/querybuf.tcl

```diff
@@ -24,8 +24,24 @@ start_server {tags {"querybuf slow"}} {
     # The test will run at least 2s to check if client query
     # buffer will be resized when client idle 2s.
     test "query buffer resized correctly" {
-        set rd [valkey_client]
+
```

Comment:
it's better to add two more cases that commands with large arguments exceed 16KB (PROTO_IOBUF_LEN) and 32KB (PROTO_MBULK_BIG_ARG), that we can cover all special cases.

Reply:
`query buffer resized correctly` checks for > 16K, the following test `query buffer resized correctly when not idle` checks for > 32K.